showcase/src/app/styleguide/pages/layout/styleguide-layout-page.component.html

```diff
@@ -0,0 +1,43 @@
+<h1 class="mat-display-3" translate>SHOWCASE.DEMO.LAYOUT.TITLE</h1>
+<h2 class="mat-display-2" translate>SHOWCASE.DEMO.LAYOUT.ANGULAR_FLEX_LAYOUT</h2>
+<p [innerHTML]="'SHOWCASE.DEMO.LAYOUT.INTRO' | translate"></p>
+
+<example-viewer filesPath="flex-layout/flex"
+				[extensions]="['HTML','SCSS']"
+				exampleTitle="SHOWCASE.DEMO.LAYOUT.FLEX">
+	<div class="layout-demo"
+		 fxLayout="column"
+		 fxLayout.gt-sm="row"
+		 fxLayoutGap="10px">
+		<div fxFlex>1</div>
+		<div class="accent"
+			 fxFlex
+			 fxFlex.gt-md="75%">
+			2
+		</div>
+		<div fxFlex>3</div>
+		<div fxFlex>4</div>
+	</div>
+</example-viewer>
+
+<div class="warning">
+	<mat-icon svgIcon="alert-circle" starkSvgViewBox></mat-icon>
+	{{ "SHOWCASE.DEMO.LAYOUT.GRID_WARNING" | translate }}
```

Comment:
I think this warning text should be put inside the `example-viewer` below otherwise the page looks a bit messy and the warning block seems to be another example block but it is not.